prow/jobs/custom/infra.yaml

```diff
@@ -48,7 +48,7 @@ presubmits:
     cluster: "prow-build"
     spec:
       containers:
-      - image: us-docker.pkg.dev/k8s-infra-prow/images/configurator:v20240405-c76de01869
+      - image: gcr.io/k8s-prow/configurator:v20240731-a5d9345e59
```

Comment:
Can we use gcr.io/k8s-staging-test-infra/configurator instead? The images were split by the project. I need to create a bumper config for that registry too